vars/getBlueoceanRestURLJob.groovy

```diff
@@ -0,0 +1,34 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Super super NIT: wdyt about renaming this file (and tests and docs) to `getBlueOceanRestURLJob.groovy`, to be consistent with `Blue Ocean` camel case?

Reply:
Unfortunately we got an old step -> `getBlueoceanDisplayURL` so I'd prefer to keep both with the same format, for backward compatibility, Though `getBlueoceanDisplayURL` might not be used inside any declarative pipeline

Reply:
Good to know, thanks for the explanation